Dockerfile.base

```diff
@@ -58,6 +58,9 @@ COPY --from=cache --chown=$UID /calypso/.nvm /calypso/.nvm
 COPY --from=cache --chown=$UID /calypso/.cache /calypso/.cache
 COPY --from=cache --chown=$UID /calypso/.bashrc /calypso/.bashrc
 
+# Copy node_modules to reduce install time of future builds.
+COPY --from=cache --chown=$UID /calypso/node_modules /calypso/wp-calypso/node_modules
```

Comment:
I think it has to be `/calypso/node_modules`, TC will overwrite `/calypso/wp-calypso` when the image is mounted.

Reply:
Gotcha. I was hoping we could avoid that, but I guess `mv` is pretty cheap.

Reply:
Actually, that's not true. From what I see in the logs, TeamCity will create a directory `/home/teamcity-3/buildAgent/work/<hash>/` inside the Docker image, that will contain `./wp-calypso` with the checkout of the repo.

So either `/calypso/node_modules` or `/calypso/wp-calypso/node_modules` will still exist in the docker image, but we need to manually move them to `${PWD}/node_modules` as part of the build steps, otherwise `node_modules` will get ignored.

Reply:
Awesome, that's really helpful